app/src/main/java/io/gymnasion/cashgroupatms/AtmAdapter.java

```diff
@@ -0,0 +1,56 @@
+package io.gymnasion.cashgroupatms;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import io.gymnasion.cashgroupatms.AtmViewHolder;
+
+public class AtmAdapter extends RecyclerView.Adapter<AtmViewHolder> {
+
+    private List<String> atmsList;
+
+
+
+    // Provide a suitable constructor (depends on the kind of dataset)
+    public AtmAdapter(List<String> atmsData) {
+        setAtmsList(atmsData);
+    }
+
+    public void setAtmsList(List<String> atmsData) {
```

Comment:
use this one in activity and add notifyDataSetChanges method call to update the ui